tests/integration/utils.js

```diff
@@ -175,6 +175,26 @@ testUtils.isCouchDB = function (cb) {
   });
 };
 
+testUtils.getServerType = function (cb) {
+  const knownServers = [
+    'couchdb',
+    'express-pouchdb',
+    'pouchdb-express-router',
+  ];
+
+  var {url, options} = parseHostWithCreds(testUtils.couchHost());
```

Comment:
```suggestion
  const { url, options } = parseHostWithCreds(testUtils.couchHost());
```

Reply:
Sorry for the delay...
Just had a rough time to get this running on my machine.

Every other test worked without a problem, so I didn't thought about any configuration problem.

But here it is: when setting the CouchDB port via couchdb.ini,
the authorization header isn't passed correctly by `PouchDB.fetch`, resulting in `401 Unauthorized` errors but `db.fetch` keep working as expected...